src/ui/public/paginated_table/paginated_table.js

```diff
@@ -25,16 +27,14 @@ uiModules
         direction: null
       };
 
-      self.sortColumn = function (colIndex) {
+      self.sortColumn = function (colIndex, sortDirection) {
```

Comment:
If it makes sense, you can set a default sortDirection on the arg: `function (colIndex, sortDirection = 'asc') {`.